target_chains/ethereum/contracts/forge-test/Pyth.WormholeMerkleAccumulator.t.sol

```diff
@@ -85,6 +85,24 @@ contract PythWormholeMerkleAccumulatorTest is
         assertEq(emaPrice.publishTime, priceFeed.emaPrice.publishTime);
     }
 
+    function assertParsedPriceFeed(
+        PythStructs.PriceFeed memory priceFeeds1,
+        PythStructs.PriceFeed memory priceFeed2
+    ) internal {
```

Comment:
There is a typo here (priceFeeds1). Can we also make it something like assertPriceFeedEqual? it is comparing two price feeds